content/en/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -14,7 +14,19 @@ You've deployed your application and exposed it via a service. Now what? Kuberne
 
 ## Organizing resource configurations
 
-Many applications require multiple resources to be created, such as a Deployment and a Service. Management of multiple resources can be simplified by grouping them together in the same file (separated by `---` in YAML). For example:
+Many applications require multiple resources to be created, such as a Deployment and a Service. Management of multiple resources can be simplified by grouping them together in the same file.
+Techniques for grouping together resources in YAML include separating:
+
+ - Different resource types with hyphens (`---`)
+ - Similar resource types as items in a list (such as `PodList`, `ServiceList`, or `SecretList`)
+ - Different resource types as items in a list (using `List`)
+
+   {{< note  >}}
+   Unlike `PodList`, `PodService`, and similar concrete list resources, the Kubernetes API server does not have a `List` resource type. Tools such as `kubectl` use `kind: List` as a way of holding a set of mixed resource items in the same YAML file or displaying mixed resources together. The API server processes each item as its own resource and does not store it as belonging to a `List` object.
```

Comment:
I'm not sure any tool other than kubectl does this.

I think this disclaimer is a little verbose but I don't have a suggestion for how to improve it, sorry :/

Reply:
Okay. I'm going to start from scratch when I cover this content in the `api-concepts` section. I'll remove everything from this file.

Reply:
Done.